lms/static/sass/views/_support.scss

```diff
@@ -132,3 +132,116 @@
     text-shadow: none;
   }
 }
+
+.contact-us-wrapper {
+
+  .row {
+    max-width: 50%;
+  }
+
+  .form-group {
+    font-family: inherit;
+  }
+
+  h2 {
+    font-size: 24px;
+    font-weight: bold;
+  }
+
+  p {
+    font-size:20px;
+    font-weight:400;
+  }
+
+  label {
+    font-family: inherit;
+    font-size: 18px;
+    font-style: normal;
+    font-weight: 400;
+  }
+
+  .help-button {
+    margin-bottom: 20px;
+    width: 160px;
+    height: 40px;
+    font-weight: 400;
+    font-size: 18px;
+    border:1px solid #0273D5;
+    border-radius:3px;
+    color:#0273D5;
```

Comment:
use color variables define https://github.com/edx/edx-platform/blob/master/lms/static/sass/partials/base/_variables.scss. For this one and all other colors.